src/yz_pb_search.erl

```diff
@@ -84,7 +84,10 @@ maybe_process(true, #rpbsearchqueryreq{index=Index}=Msg, State) ->
 				R = mochijson2:decode(Body),
 				Resp = yz_solr:get_response(R),
 				Pairs = yz_solr:get_doc_pairs(Resp),
-				MaxScore = kvc:path([<<"maxScore">>], Resp),
+				MaxScore = case kvc:path([<<"maxScore">>], Resp) of
```

Comment:
Realizing we'll need to update kvc to a new tag to make this happen. I'm taking a look into it.